demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -359,6 +360,15 @@ def get_duplicate_pack_display_name(
             )
             return results
 
+    def get_duplicate_script_name_included_incident(
+        self, file_paths: List[str]
+    ) -> Dict[str, str]:
+        with self.driver.session() as session:
+            results = session.execute_read(
+                validate_multiple_script_with_same_name, file_paths
+            )
+        return results
```

Comment:
```suggestion
            return session.execute_read(
                validate_multiple_script_with_same_name, file_paths
            )
```
python closes the scope anyway